test/acceptance/log.provider.test.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Client, createClientForApp, sinon} from '@loopback/testlab';
+import {inject} from '@loopback/context';
+import {
+  Application,
+  get,
+  param,
+  SequenceHandler,
+  FindRoute,
+  ParseParams,
+  InvokeMethod,
+  Send,
+  Reject,
+  CoreBindings,
+  ParsedRequest,
+  ServerResponse,
+} from '@loopback/core';
+import {LogComponent, ExtensionStarterBindings, LogFn} from '../..';
+
+const coreSequenceActions = CoreBindings.SequenceActions;
+
+describe('Logging', () => {
```

Comment:
`describe('Logging (acceptance')`.

Also the test file should be `test/acceptance/providers/log-action.provider.acceptance.ts`.